lhotse/cut.py

```diff
@@ -2184,6 +2293,45 @@ def truncate(
 
         return new_cut
 
+    def extend(
```

Comment:
I don't believe this method is well defined on `MixedCut`, TBH I'd just `raise ValueError("It is undefined to extend() a MixedCut.")`